src/main/java/rs/teslaris/core/service/impl/document/BookSeriesServiceImpl.java

```diff
@@ -175,6 +175,8 @@ private void indexCommonFields(BookSeries bookSeries, BookSeriesIndex index) {
     }
 
     private Query buildSimpleSearchQuery(List<String> tokens) {
+        var minShouldMatch = (int) Math.ceil(tokens.size() * 0.8);
```

Comment:
this might be ok for title of an article, but not sure for person names is perfect, I think if we have three words in name, it might be enough to have matching for two of them to be a candidate which should be listed. However, let's start with this, but it might be that it is not going to be the same number in all cases (0.8)

Reply:
in the current CRIS UNS it is not the same in all cases.

Reply:
should I switch to 0.75 for person name?

Reply:
I would suggest to 0.65

Reply:
sorry, let's keep as is at the moment, we will see how it is working in practice, it will be easy to change later, it is important this coefficient can be defined separately for person name